packages/clerk-js/src/core/clerk.ts

```diff
@@ -1967,10 +1971,19 @@ export class Clerk implements ClerkInterface {
     if (!key || !this.loaded || !this.environment || !this.environment.displayConfig) {
       return '';
     }
+
     const signInOrUpUrl = this.#options[key] || this.environment.displayConfig[key];
+    const combinedFlowSignUpUrl = `${this.#options.signInUrl}#/create`;
+    const baseUrl = this.isCombinedFlow() && key === 'signUpUrl' ? combinedFlowSignUpUrl : signInOrUpUrl;
```

Comment:
Maybe need to also check for experimental combinedFlow flag here too?

Reply:
Yeah that would be good, just to be sure. It's at the component level now, not at the core Clerk level. 👀